swap/swap_test.go

```diff
@@ -88,6 +91,15 @@ func init() {
 	testBackend = newTestBackend()
 	// commit the initial "pre-mined" accounts (issuer and beneficiary addresses)
 	testBackend.Commit()
+
+	var err error
+	//Swap log default path
```

Comment:
i don't understand this comment, can you please clarify?

Reply:
fixed